entrypoint/fb.js

```diff
@@ -106,3 +110,64 @@ module.exports.message = (event, context, callback) => {
     chat.sendText('Da ist was schief gelaufen.');
   });
 };
+
+module.exports.push = (event, context, callback = console.log) => {
+  let params = null;
+
+  if (event.queryStringParameters !== undefined) {
+    params = event.queryStringParameters;  // Called from web
+  } else {
+    params = event;  // Called from scheduler
+  }
```

Comment:
This breaks if the http function is called without queryStringParameters. Should rather be determined by `event['detail-type'] == "Scheduled Event"`

Reply:
More Info https://docs.aws.amazon.com/de_de/lambda/latest/dg/eventsources.html#eventsources-scheduled-event

Reply:
Or some other method to determine if called via http